lib/publiccloud/provider.pm

```diff
@@ -260,10 +260,7 @@ sub terraform_apply {
     my @instances;
     my $create_extra_disk = 'false';
     my $extra_disk_size   = 0;
-    if ($args{use_extra_disk}) {
-        $create_extra_disk = 'true';
-        $extra_disk_size   = $args{use_extra_disk}->{size};
-    }
+    my $extra_disk_type   = undef;
```

Comment:
we actually not using this variable anymore so it can be deleted

Reply:
done